server/store.go

```diff
@@ -86,13 +86,18 @@ func (l *listStore) GetIssue(issueID string) (*Issue, error) {
 	return issue, nil
 }
 
-func (l *listStore) RemoveIssue(issueID string) error {
+func (l *listStore) RemoveIssue(issueID string) (*Issue, error) {
+	issue, err := l.GetIssue(issueID)
```

Comment:
nit: I question this pattern (a mandatory Get before removing) but only because of the name of this function. Not sure how to name it better though.

Reply:
Looking through its use, I think you should have 2 functions - one with a get, and one without. Or maybe just `Get` explicitly where needed? Seems like a handful of places.